src/wp-includes/media.php

```diff
@@ -2029,6 +2029,22 @@ function wp_sizes_attribute_includes_valid_auto( string $sizes_attr ): bool {
 	return 'auto' === strtolower( trim( $first_size, " \t\f\r\n" ) );
 }
 
+/**
+ * Prints a CSS rule to fix potential visual issues with images using `sizes=auto`.
+ *
+ * This rule overrides the similar rule in the default user agent stylesheet, to avoid images that use e.g.
+ * `width: auto` or `width: fit-content` to appear smaller.
+ *
+ * @since 6.7.1
+ * @see https://html.spec.whatwg.org/multipage/rendering.html#img-contain-size
+ * @see https://core.trac.wordpress.org/ticket/62413
+ */
+function wp_print_auto_sizes_contain_css_fix() {
+	?>
+	<style>img:is([sizes="auto" i], [sizes^="auto," i]) { contain-intrinsic-size: 3000px 1500px }</style>
```

Comment:
Per https://core.trac.wordpress.org/ticket/62413#comment:39 browser adds `contain: size !important;` CSS rule do we needs add it also?
```
img:is([sizes="auto" i], [sizes^="auto," i]) {
    contain: size !important;
    contain-intrinsic-size: 300px 150px;
}
```

Reply:
We only need `contain-intrinstic-size`, since that's what causes the problem.